### ISSUE TITLE
Should allow a way to boot a u-boot binary with serial completely disabled

### ISSUE THREAD

**Issue by ogra1** [snapcore/pi-gadget#47]
Many addon boards for the Pi talk to the host system by being attached to the serial port header. Often these boards get powered by the mains power so that they are up and listening on the serial connection before u-boot comes up ... 

If u-boot writes *anything* to the serial console in this combination the addon board typically go into a bad state. To use such boards on Ubuntu Core a way to boot with a completely quiet u-boot on serial should be provided. Preferably via an easy to set (and well documented) option or environment variable when building the gadget.

**Comment by bencorrado** [comment]
I completely agree with @ogra1 on this.  Currently Wirepas users wanting to use Core with a RPI like to use this board: https://store.clemanis.com/en/iot/1187-raspberry-hat.html  Having a printout at boot on the UART confuses the system.

**Comment by demison** [comment]
RPI4 with RaZberry card (z-wave interface card) does not start at all. No boot logo, no logs. Hangs on start.
I assume it's the same issue.
P.S. Works like a charm with same hardware, Raspberry Pi OS and disabled bluetooth.

**Comment by waveform80** [comment]
Resolved with the removal of u-boot from the boot chain in Core 24
